unit_tests/test_version_utils.py

```diff
@@ -397,7 +397,7 @@ def test_scylla_version_grouped_regexp(full_version, version, date, commit_id):
 @pytest.mark.parametrize("version, expected", (
     ("4.6.rc2-20220102.e8a1cfb6f", "scylladb/scylla:4.6.rc2"),
     ("5.0.1-0.20220719.b177dacd3", "scylladb/scylla:5.0.1"),
-    ("5.1", "scylladb/scylla:latest"),
+    ("5.1", "scylladb/scylla:5.1"),
```

Comment:
Need to add following:

```
("5.2", "scylladb/scylla:latest"),
```

Reply:
why ? for it to fail once they would start the 5.2 release ?
I think this is a bit pointless

Reply:
I know that we will hit the same problem again, but it is coverage of the functionality.
We either test that the next version is refering to "latest" one or not.

Reply:
right now 5.2 doesn't exist, we already have a test covering that.

I'm don't understand what adding this new test would help us ?

Reply:
It is for testing mapping of next unavilable version yet and `latest` tag.
If you think we do not need then ok.